src/pages/fundamentals/eutxo.mdx

```diff
@@ -354,9 +353,9 @@ verification key hash digest or as script hash digest.
 
 Before we move on, let's recap again:
 
-- An _address_ is made of 2 or 3 parts: a _header_, _payment credentials_ and _delegation credentials_;
```

Comment:
Prefer 'is made or 2 or 3'.

Reply:
Yes, I noticed this style is used in a few places but this reads a little weird imo (native speakler).  Perhaps "either two or three parts"? 
(I also have throughout ignored that the usual form in English is that numbers ten and below are usually kept as words, for example)

Reply:
typo* is made of 2 or 3

Reply:
Micah is also a native speaker

Reply:
Technical jargon alters from professional English. 2,3 are technical values.

Reply:
ya

Reply:
Edited to: 
"An address is made up of either 2 or 3 parts: it always contains a header and payment credentials, and may optionally also contain delegation credentials;"

Reply:
> (especially UK rules) 
No need for racism, this was just an example (the grammatical rules I was talking about are true to "international" English)

Reply:
> (especially UK rules) 
No need for racism, this was just an example (the grammatical rules I was talking about are true to "international" English)

Reply:
> (especially UK rules) 
No need for racism, this was just an example (the grammatical rules I was talking about are true to "international" English)